staging/src/k8s.io/kubectl/pkg/completion/utils.go

```diff
@@ -0,0 +1,90 @@
+/*
```

Comment:
For reviewer: I move helper functions here as I think they might be reused for the completion of `kubectl config get-{contexts,clusters}`